doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/azure.md

```diff
@@ -0,0 +1,83 @@
+
+# Launching Ray Clusters on Azure 
+
+This guide details the steps needed to start a Ray cluster on Azure.
+
+There are two ways to start an Azure Ray cluster.
+- Launch through Ray Cluster Launcher.
+- Deploy a cluster using Azure portal .
```

Comment:
Do we officially recommend this approach? I read this subsection and did not really understand it.

Reply:
Yeah I'm mostly copy over the content over. cc @DmitriGekhtman do you know if this really used?

Reply:
The azure thing is fully community maintained. I've added a note at top